src/utils/getMaxId.js

```diff
@@ -0,0 +1,13 @@
+function getCreateMaxId(array) {
+  if (array.length === 0) {
+    return 1;
+  } else {
+    const maxId = Math.max(...array.map((obj) => obj.id));
+
+    return maxId + 1;
```

Comment:
You've added the `uuid` library, you can use it to generate unique id

Reply:
At first I made a uuid library, but the tests skip only digits.

Reply:
> At first I made a uuid library, but the tests skip only digits.

Ok, so remove the `uuid` package if you don't use it